README.md

```diff
@@ -23,6 +23,17 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
+Register at [ReCAPTCHA](http://www.google.com/recaptcha/admin) using the following configuration.
+|Field       |Value      |
+|--------------|-----------|
+|Label         |short      |
+|reCAPTCHA type|reCAPTCHAv2|
```

Comment:
I wasn't sure if v2 or v3 was needed so I picked v2. Can you verify which one is needed please!

Reply:
We use V3

Reply:
Updated.